src/views/Graph.vue

```diff
@@ -622,14 +623,15 @@ export default {
         await this.$nextTick()
       }
     },
+    /** re-layout the graph after any new nodes have been rendered */
     async layout (nodes, edges, nodeDimensions) {
-      // re-layout the graph after any new nodes have been rendered
+      const graphviz = await Graphviz.load()
```

Comment:
What are the implications of calling `Graphviz.load` for every layout?

Could we do this on mount?